autogen.sh

```diff
@@ -10,5 +10,7 @@ fi
 
 # old autoreconf/aclocal versions fail hard if m4 doesn't exist
 mkdir -p m4
+# some versions fail if it doesn't exist
+touch ./config.rpath
```

Comment:
Is this the right thing to do?  Would you provide more details?  (autoconf can be quite opaque and obtuse.  lighttpd does include some m4 scripts to address some shortcomings or limitations in some distributions.  Is there a better way to get past the error you are seeing?  When are you seeing an error?  What versions/platforms?)

Related, lighttpd supports multiple build systems, so if we add --with-wolfssl to configure.ac, we should also update the CMake, SCons, and meson build scripts, too.